packages/snaps-cli/README.md

```diff
@@ -21,6 +21,7 @@ Commands:
   mm-snap build     Build snap from source                          [aliases: b]
   mm-snap eval      Attempt to evaluate snap bundle in SES          [aliases: e]
   mm-snap manifest  Validate the snap.manifest.json file            [aliases: m]
+  mm-snap sandbox   Start a sandbox server to interact with the Snap
```

Comment:
No alias for this one ?

Reply:
`s` was already taken, and I'm not sure what else to use 😅

Reply:
`x`?